openmetadata-ui/src/main/resources/ui/src/pages/LogsViewer/LogsViewer.component.test.tsx

```diff
@@ -0,0 +1,64 @@
+/*
+ *  Copyright 2022 Collate
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import { act, render, screen } from '@testing-library/react';
+import React from 'react';
+import LogsViewer from './LogsViewer.component';
+import { mockIngestionPipeline, mockLogsData } from './mocks/LogsViewer.mock';
+
+jest.mock('react-router-dom', () => ({
+  useParams: jest.fn().mockReturnValue({
+    logEntityType: 'TestSuite',
+    ingestionName: 'ingestion_123456',
+  }),
+}));
+
+jest.mock(
+  '../../components/common/title-breadcrumb/title-breadcrumb.component',
+  () => () => <>TitleBreadcrumb.component</>
+);
+
+jest.mock('react-lazylog', () => ({
+  LazyLog: jest.fn().mockImplementation(() => <div>LazyLog</div>),
+}));
+
+jest.mock('react-i18next', () => ({
+  useTranslation: jest.fn().mockReturnValue({
+    t: (key: string) => key,
+  }),
+}));
+
+jest.mock('../../axiosAPIs/ingestionPipelineAPI', () => ({
+  getIngestionPipelineLogById: jest
+    .fn()
+    .mockImplementation(() => Promise.resolve(mockLogsData)),
+  getIngestionPipelineByName: jest
+    .fn()
+    .mockImplementation(() => Promise.resolve(mockIngestionPipeline)),
+}));
+
+describe('LogsViewer.component', () => {
+  it('On initial, component should render', async () => {
+    await act(async () => {
+      render(<LogsViewer />);
+
+      expect(
+        await screen.findByText('TitleBreadcrumb.component')
+      ).toBeInTheDocument();
```

Comment:
Increase tests cases like:

1. Should render logs view
2. should hit API to fetch logs
3. Should render search